src/test-run/index.js

```diff
@@ -422,8 +435,8 @@ export default class TestRun extends Session {
         var assertionTimeout = command.options.timeout === void 0 ? this.opts.assertionTimeout : command.options.timeout;
         var executor         = new AssertionExecutor(command, assertionTimeout, callsite);
 
-        executor.once('start-assertion-retries', timeout => this._enqueueCommand(new ShowAssertionRetriesStatusCommand(timeout)));
-        executor.once('end-assertion-retries', success => this._enqueueCommand(new HideAssertionRetriesStatusCommand(success)));
+        executor.once('start-assertion-retries', timeout => this.executeCommand(new ShowAssertionRetriesStatusCommand(timeout)));
```

Comment:
We should not add ` this.debugLog.command(command);` for these commands (see [line](https://github.com/churkin/testcafe/blob/19ab20585dd46cf79456f6755e1e3b604b846a7e/src/test-run/index.js#L485))

Reply:
Why? We have discussed with @AndreyBelym and have decided that should.

Reply:
Yes, we should, I mix up different methods